cellmlmanip/units.py

```diff
@@ -79,6 +79,77 @@
 }
 
 
+class UnitError(Exception):
+    """ base class for unit errors
```

Comment:
I'd prefer it if we added capitalisation and punctuation in docstrings, so that we can easily generate nice API documentation later (e.g. via readthedocs.io, like here: https://pints.readthedocs.io/en/latest/ )

Reply:
Done for units.py which is the main file I touched here. Will do a check as I hit other files.

Reply:
Thanks so much! I'll make an issue for the other ones